boule/sphere.py

```diff
@@ -193,6 +193,13 @@ def normal_gravity(self, latitude, height, si_units=False):
             The normal gravity in mGal.
 
         """
+        # Warn if height is negative
+        if np.any(height < 0):
+            warn(
+                "Formulas used are valid for points outside the ellipsoid."
+                "Height must be major or equal to zero."
+            )
+
```

Comment:
We probably don't need to leave this empty line, the method is quite short so we don't need to split its code for now.
```suggestion
```